<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1157</width>
    <height>724</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1141</width>
      <height>661</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#E4E4E4;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>Соединение</string>
     </attribute>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>275</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_create">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>360</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:10px;
background-color:gray;
color:white;</string>
      </property>
      <property name="text">
       <string>Создать</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_our_port">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>300</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Наш порт</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_port">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>200</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
border:1px solid blue;
border-radius:10px;
padding:2px;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_port">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>200</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Порт</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_our_port">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>300</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
border:1px solid blue;
border-radius:10px;
padding:2px;
</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_adress">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>250</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
border:1px solid blue;
border-radius:10px;
padding:2px;
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_adress">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>250</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Адрес</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_our_adress">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>410</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Сообщения</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_send">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>560</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:10px;
background-color:gray;
color:white;</string>
      </property>
      <property name="text">
       <string>Отправить</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_input">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>40</y>
        <width>421</width>
        <height>511</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="tabChangesFocus">
       <bool>false</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_message">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>560</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
border:1px solid blue;
border-radius:10px;
padding:2px;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_seans">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>10</y>
        <width>411</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Reference Sans Serif</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>560</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
border:1px solid blue;
border-radius:10px;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>470</y>
        <width>241</width>
        <height>81</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>241</width>
        <height>81</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>241</width>
        <height>81</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
border-radius:10px;
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButton7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;
border:1px solid blue;
border-radius:10px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton12">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>40</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;
border:1px solid blue;
border-radius:10px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton11">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>40</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;
border:1px solid blue;
border-radius:10px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton10">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;
border:1px solid blue;
border-radius:10px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>0</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;
border:1px solid blue;
border-radius:10px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton6">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>0</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;
border:1px solid blue;
border-radius:10px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton4">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>0</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;
border:1px solid blue;
border-radius:10px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton9">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;
border:1px solid blue;
border-radius:10px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton3">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>0</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;
border:1px solid blue;
border-radius:10px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;
border:1px solid blue;
border-radius:10px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton5">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>0</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;
border:1px solid blue;
border-radius:10px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton8">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;
border:1px solid blue;
border-radius:10px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <zorder>pushButton_send</zorder>
     <zorder>lineEdit_message</zorder>
     <zorder>label_seans</zorder>
     <zorder>pushButton</zorder>
     <zorder>textEdit_input</zorder>
     <zorder>frame</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1157</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
